enterprise/frontend/src/embedding-sdk/hooks/private/use-load-static-question.ts

```diff
@@ -60,24 +50,24 @@ export function useLoadStaticQuestion(
           cancelDeferred,
         });
 
-        setQuestionState({
-          card,
-          result,
-          loading: false,
-          error: null,
-        });
-      } catch (error) {
-        if (!isMounted.current) {
-          return;
-        }
-
-        if (typeof error === "object") {
+        if (!ignore) {
           setQuestionState({
-            result: null,
-            card: null,
+            card,
+            result,
             loading: false,
-            error,
+            error: null,
           });
+        }
+      } catch (error) {
+        if (typeof error === "object") {
+          if (!ignore) {
```

Comment:
This can't be added as `&& !ignore` otherwise we'd go in the else -> log the (useless) error every time the component mounts in strict mode